index.html

```diff
@@ -233,5 +233,78 @@ <h1 class="about_myself_heading">About me</h1>
         </li>
       </ul>
     </section>
+
+    <section class="contact_section">
+      <h3>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+
+      <form
+        action="https://formspree.io/f/xgedowvw"
+        method="POST"
+        class="contact_form"
+      >
+        <input
+          class="name"
+          type="text"
+          placeholder="Full name"
+          required
+          maxlength="30"
+        />
+        <hr class="horizontal_line" />
+        <input
+          class="email"
+          type="email"
+          placeholder="Email address"
+          required
+        />
+        <hr class="horizontal_line" />
+        <textarea
+          class="message"
+          name="message"
+          id="message"
+          cols="30"
+          rows="10"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
```

Comment:
Addressed 🔥😎🙌

Reply:
Thank you @mateo951 for reviewing my project.

Reply:
Congrats Zel 👍